modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/service/impl/RemoteAppEntryLocalServiceImpl.java

```diff
@@ -83,6 +88,23 @@
 public class RemoteAppEntryLocalServiceImpl
 	extends RemoteAppEntryLocalServiceBaseImpl {
 
+	@Override
+	public FileEntry addAttachment(
```

Comment:
Do we already know what we need to do to make files appear in every node when clustering is enabled? We have discussed it in the past, but I don't know if we have arrived to a conclusion...

Reply:
I'm trying to solve this need, but I have no experience with it, so I'm sure I can miss some considerations or old discussions

Reply:
Put this on hold and try to pass a first PR without attachments